### PR TITLE
Fixes #3196 - Add functional tests for the new form UI

### LINKED ISSUE #3196: Add functional tests for the new form UI
We need to add tests for the new form, similar to https://github.com/webcompat/webcompat.com/blob/master/tests/functional/reporting-non-auth.js

### PR DESCRIPTION
**Pull request by ksy36**
r? @miketaylr

### CODE REVIEW

File: webcompat/templates/issue-wizard-form.html [hunk 1]
```diff
@@ -371,6 +371,7 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
                     name="github-proxy-report">
               Send report
             </button>
+           </div>
```
Comment (miketaylr) [inline webcompat/templates/issue-wizard-form.html]: 👻

File: tests/functional/reporting-issue-wizard-non-auth.js [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+        })
+        .end()
+        .findByCssSelector("#url")
+        .clearValue()
+        .type("http://sup.com")
+        .end()
+        .waitForDeletedByCssSelector(".form-message-error")
+        .end();
+    },
+    "Wizard stepper - scenario 1"() {
```
Comment (miketaylr) [inline tests/functional/reporting-issue-wizard-non-auth.js]: Could you maybe add comments that describe what the 3 scenarios are?

**Comment by ksy36** [comment]
Good point, thanks :)

**Comment by ksy36** [comment]
Got 6 continuous jobs successfully passing, so I think it's good to go. @miketaylr could you please take a look one more time?

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**